tests/unit/s2n_client_hello_retry_test.c

```diff
@@ -959,6 +959,9 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_set_server_name(client_conn, "localhost"));
             EXPECT_SUCCESS(s2n_connection_append_protocol_preference(client_conn, apn, sizeof(apn)));
             EXPECT_SUCCESS(s2n_connection_set_early_data_expected(client_conn));
+            client_conn->handshake.renegotiation = true;
+            client_conn->secure_renegotiation = true;
```

Comment:
This shouldn't be necessary? There's an exception for the renegotiation_info extension in this test.